int32 category_id
string category_name
sensor_msgs/RegionOfInterest bounding_box
sensor_msgs/Image mask
float32 score
